app/models/automation/webhook.rb

```diff
@@ -0,0 +1,32 @@
+# == Schema Information
+#
+# Table name: automation_webhooks
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  url        :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#  tenant_id  :bigint           not null
+#
+module Automation
+  class Webhook < ApplicationRecord
+    belongs_to :tenant
+    has_many :automation_actions, class_name: "Automation::Action", as: :action_object, dependent: :restrict_with_error
+
+    validates_presence_of :name, :url
+
+    def fire!(message, event, timestamp, downloader: Faraday)
+      data = {
+        type: "#{message.class.name}.#{event}",
```

Comment:
Tu by som mozno pridal `underscore` nech drzime konvencie rails/ruby

Reply:
podľa toho štandardu má byť bodka

Reply:
to chapem, ale nazov triedy je camecase a vsetko ostatne je underscore

Reply:
jo, už rozumiem. Dal som .underscore